README.md

```diff
@@ -106,31 +111,31 @@ Which one of the following concepts does this best illustrate?
 ```
 [ ] A: Modularity
 [ ] B: MVC
-[ ] C: Two-way data-binding
+[X] C: Two-way data-binding
 [ ] D: Separation of concerns
 ```
 
 ## Question 6
 
 What is an IIFE, and why might you use it?
 
-**Your answer:**
+IIFE stands for Immediately-Invoked Function Expression and it defined as a function that calls itself right when it's defined but is then not used again.
 
 > ...
 
 ## Question 7
 
 What is the `ui-sref` directive, and how is it used?
 
-**Your answer:**
+It is used to generate links in angular.
 
```

Comment:
👍 a directive given to us by ui-router that allows us to jump to a different state